samples/samples-aspnet/embedded-sign-in-widget/README.md

```diff
@@ -0,0 +1,93 @@
+
+> Because of [Set-Cookie behavior (SameSite)](https://web.dev/samesite-cookies-explained) this code will only work properly if it's configured to use https. See [Work with SameSite cookies in ASP.NET Core](https://docs.microsoft.com/en-us/aspnet/core/security/samesite?view=aspnetcore-3.1) for more details.
+
+#### Run the web application from Visual Studio
+
+When you run this project in Visual Studio it starts the web application on port 44314 using HTTPS. You can change this configuration in the `launchSettings.json` in the Properties folder.
+
+#### Run the web application from dotnet CLI
+
+When you run this project via the dotnet CLI it will start the web application on port 5001 using HTTPS. You can change this configuration in the `launchSettings.json` in the Properties folder.
+
+Navigate to the folder where the project file is located and type the following:
+
+```dotnet run```
+
+### Trust the local dev certificate if necessary
+
+When you run an ASP.NET Core 3.x application for the first time, you may notice an error page warning you that the site is potentially unsafe.
+This is because ASP.NET Core creates an HTTPS development certificate for you as part of the first-run experience that needs to be trusted by your browser. You can ignore the warning by clicking on Advanced and telling the browser to visit this site even though there is no certificate for it. Or you can trust the certificate to get rid of this warning, check out [Configuring HTTPS in ASP.NET Core across different platforms] for more details.
+
+### Configure your application in the Okta Developer Console
+
+Go to your [Okta Developer Console] and update the following parameters in your Okta Web Application configuration:
+* **Login redirect URI** - `https://localhost:44314/interactioncode/callback`
+* **Logout redirect URI** - `https://localhost:44314/account/signout`
+
+The sample application defines an `InteractionCodeController` which receives the `interaction_code` upon successful login; review the `RedeemInteractionCodeAndSignInAsync` method of the `InteractionCodeController` for example code illustrating how to exchange the interaction code for Okta tokens.
+
+### Enable CORS (Cross-Origin Resource Sharing)
+
+Your application must be configured to allow your application to make requests to the Okta API using the Okta session cookie. To enable CORS for your application do the following:
+
+- In your [Okta Developer Console], go to **Security > API > Trusted Origins** 
+- Add your web application’s base URL `https://localhost:44314/` as a **Trusted Origin**.
+
+For step-by-step instructions, visit the Okta [Sign Users in to Your Web Application guide] which shows how to sign users in using Okta and, [Sign Users Out guide] which shows how to sign users out of your application and out of Okta.
+
+### Configure your project
+
+Replace the okta configuration placeholders in the `appsettings.json` with configuration values from your [Okta Developer Console]. 
```

Comment:
Is this section relevant? Aren't we using the okta.yaml configuration approach here?